src/main/java/frc/robot/commands/command_groups/ShortCollectCell.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.commands.command_groups;
+
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.StartEndCommand;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.subsystems.intake.SetIntakeSpeed;
+import frc.robot.subsystems.intakeopener.SetIntakeState;
+import frc.robot.subsystems.led.LEDColor;
+
+import static frc.robot.Robot.led;
+import static frc.robot.Robot.robotConstants;
+
+public class ShortCollectCell extends SequentialCommandGroup {
+
+    private static final int kCollectTime = 2;
+
+    public ShortCollectCell() {
+        addCommands(
+            new StartEndCommand(() -> led.setColor(LEDColor.Orange), () -> led.turnOffLED(), led),
+            parallel(
+                new SetIntakeState(true),
+                new SetIntakeSpeed(robotConstants.intakeConstants.kDefaultIntakePower)
```

Comment:
not finished, also - new SetIntakeSpeed(robotConstants.intakeConstants.kDefaultIntakePower)
use regular constructor